scrapers/Andomark.yml

```
@@ -34,9 +74,73 @@ xPathScrapers:
+                      with: "" 
+                - replace:
+                    - regex: https://(w{3}?\.?)?(.+?)(\..+)
                       with: $2
-      Image: //meta[@property="og:image"]/@content
+                - map:
+                   "ariellynn": Ariel Lynn
+                   "beatmeup": Beatmeup
+                   "behindtheporno": Behind The Porno
+                   "bigboobiesclub": Big Boobies Club
+                   "bigbouncybabes": Big Bouncy Babes
+                   "bigtoyxxx": Big Toy XXX
+                   "bradsterling": Brad Sterling XXX
+                   "britstudio": Brit Studio
+                   "brittanyandrewsxxx": BritttanyAndrews XXX
+                   "chocolatepov": ChocolatePOV
+                   "collectivecorruption": Collective Corruption
+                   "datingapphookup": Dating App Hook Ups
+                   "dirtroadwarriors": Dirt Road Warriors
+                   "goonmuse": GoonMuse
+                   "internationalnudes": International Nudes
+                   "hollyhotwife": HollyHotWifeXXX
+                   "houseofyre": House of Fyre
+                   "johnnygoodluck": Johnny Goodluck
+                   "justgoodsex": JustGoodSex
+                   "ravenswallowzxxx": Raven Swallows
+                   "reidmylips": Reid My Lips
+                   "sheseducedme": She Seduced Me
+                   "mackmovies": Mack Movies
+                   "melaniehicksxxx": Melanie Hicks XXX
+                   "oldsexygrannies": OldSexyGrannies
+                   "playpetvictoria": PlaypetVictoria
+                   "queercrush": QueerCrush
+                   "rionkingxxx": Rion King
+                   "seanmichaelsxxx": Sean Michaels
+                   "sofiemariexxx": SofieMarieXXX
+                   "tabooadventures": TabooAdventures
+                   "taboovrporn": TabooVRPorn
+                   "terapatrick": Terra Patrick
+                   "texaspattiusa": TexasPattiUSA
+                   "tmfetish": TMFetish
+                   "vanillapov": VanillaPOV
+                   "willtilexxx": WillTileXXX
+                   "xevunleashed": Xev Unleashed
+                   "xxxcellentadventures": XXXcellentAdventures
+                   "younggunsxxx": YoungGunsXXX
+                   "yummybikinimodel": YummyBikini
+                   "yummygirl": Yummygirl
+                   "yummygirlz": Yummygirlz
+                   "yummypornclub": YummyPornClub
+                   "yummysexclub": Spik Irons' YummySexClub
+                   "yummysofie": Yum Sofie
+                   "yummywomen": Yummywomen
+      Image: 
+        selector: //base/@href | //div[@class="update_image"]/a/img/@src0_1x | //span[@class="model_update_thumb"]/img/@src
+        concat: /
+        postProcess:
+          - replace:
+              - regex: ^(https://.+)?https://
+                with: https://
       URL: //link[@rel='canonical']/@href
-
-# Last Updated October 09, 2020
+driver:
```

And here we can add a scraper for the sites that use a slight variant of the layout
```suggestion
  proSceneScraper:
    common:
      # We need to exclude anything inside a "category_listing_block" because those are suggested content and would give us false positives
      $excludeUpdates: not(ancestor::*[contains(@class, "category_listing_block")])
    scene:
      Title: //div[@class="title_bar"]
      Date:
        selector: //div[contains(@class, "update_date") and $excludeUpdates and contains(., "/")]
        postProcess: *datePostProcess
      Details: //span[@class="update_description"]
      Performers:
        Name: //span[@class="update_models" and $excludeUpdates]/a
      Tags:
        Name: //span[@class="update_tags" and $excludeUpdates]/a
      Studio:
        selector: //base/@href
        postProcess: *studioPostProcess
      Image:
        selector: //base/@href | //script[contains(text(), 'useimage = "/')]/text()
        concat: __SEPARATOR__
        postProcess:
          - replace:
              - regex: (https://[^/]+).*useimage\s*=\s*"([^"]*).*
                with: $1$2
      URL: //link[@rel='canonical']/@href

driver:
```